sim/play.py

```diff
@@ -219,16 +260,55 @@ def play(args: argparse.Namespace) -> None:
             if num_episodes > 0:
                 env_logger.log_rewards(infos["episode"], num_episodes)
 
+        if args.log_krec:
+            frame = krec.KRecFrame(
+                video_timestamp=start_time_ns + int(t * env.dt * 1e9),  # Convert simulation time to real time
+                frame_number=t,
+                inference_step=t // env_cfg.control.decimation,
+            )
+
+            # Add actuator states and commands for each joint
+            for i in range(env.num_dof):
+                state = krec.ActuatorState(
+                    actuator_id=i,
+                    online=True,
+                    position=env.dof_pos[robot_index, i].item(),
+                    velocity=env.dof_vel[robot_index, i].item(),
+                    torque=env.torques[robot_index, i].item(),
+                )
+                command = krec.ActuatorCommand(
```

Comment:
If you want to have 1to1 mapping the command information is missing here which is fundamental.